README.md

```diff
@@ -21,13 +21,13 @@ $ npm install --save blockmap
 For detailed API documentation, see [`/doc`](https://github.com/resin-io-modules/blockmap/tree/master/doc).
```

Comment:
I guess the URL here ought to be updated to balena ?

Reply:
updated